src/index.js

```diff
@@ -1,10 +1,40 @@
 const { Telegraf } = require("telegraf");
 require("dotenv").config();
+const Groq = require("groq-sdk");
```

Comment:
**praise**: Good job on importing the Groq SDK. This is essential for integrating the LLM model.